src/state/GlobalState.tsx

```diff
@@ -204,7 +206,7 @@ export class GlobalState extends PureComponent<Props> {
       this.setState({ libraryStatus: updatedLibraryStatus })
       notify([title, t('notify.uninstalled')])
 
-      return this.refresh()
+      return this.refreshLibrary()
```

Comment:
Missing `bool` argument? Or inferred `false`?

Reply:
The argument is optional. So it will evaluate to false.